wireless_modules/wireless_module.py

```diff
@@ -138,6 +138,11 @@ async def wait_for_start(self):
             # Start message received, tell sensors to start
             for sensor in self.sensors:
                 sensor.on_start()
+        
+        #Fix for changing LED when a true is retained
+        if self.start_publish and self.status_led.state != WmState.Publishing:
+            self.status_led.set_state(WmState.Publishing)
```

Comment:
Thought of an alternative: we can set the led state in `sub_cb`, so whoever we receive start message we set led in that method itself. WDYT?

Reply:
Yeah, this would be much better. That way if we received a retained topic we would automatically switch the LED rather than checking in the `wait_for_start` function, right?

Reply:
Yeah exactly. We can then remove `self.status_led.set_state(WmState.Publishing)` and `self.status_led.set_state(WmState.Idle)` from `wait_for_start()` and move them to `sub_cb()`